batch7crud-roll3/src/main/java/com/lftechnology/batch7crud/controller/CustomHttpServlet.java

```diff
@@ -1,16 +1,20 @@
 package com.lftechnology.batch7crud.controller;
 
-import java.io.IOException;
-import java.util.logging.Level;
+import static com.lftechnology.batch7crud.constant.CommonConstant.CREATE;
+import static com.lftechnology.batch7crud.constant.CommonConstant.DELETE;
+import static com.lftechnology.batch7crud.constant.CommonConstant.EDIT;
+import static com.lftechnology.batch7crud.constant.CommonConstant.LIST;
+import static com.lftechnology.batch7crud.constant.CommonConstant.PAGE_NOT_FOUND;
+import static com.lftechnology.batch7crud.constant.CommonConstant.PAGE_NUMBER;
+import static com.lftechnology.batch7crud.constant.CommonConstant.SHOW;
+
 import java.util.logging.Logger;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.logging.Logger'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)